File: site/code-snippets/Voice/modifyConferenceMember/sample.py
from bandwidth.bandwidth_client import BandwidthClient
from bandwidth.voice.models.conference_member_state import ConferenceMemberState
from bandwidth.exceptions.api_exception import APIException

import os

BW_USERNAME = os.environ["BW_USERNAME"]
BW_PASSWORD = os.environ["BW_PASSWORD"]
BW_ACCOUNT_ID = os.environ["BW_ACCOUNT_ID"]

bandwidth_client = BandwidthClient(
    voice_basic_auth_user_name=BW_USERNAME,
    voice_basic_auth_password=BW_PASSWORD
)
voice_client = bandwidth_client.voice_client.client

body = ConferenceMemberState()
body.mute = True

conference_id = "conf-1234"
call_id = "c-1234"

try:
    voice_client.modify_conference_member(BW_ACCOUNT_ID, conference_id, call_id, body)
except APIException as e:
    print(e.response_code)

File: site/code-snippets/Voice/getConferenceRecording/sample.php
<?php

require "vendor/autoload.php";

$BW_USERNAME = getenv("BW_USERNAME");
$BW_PASSWORD = getenv("BW_PASSWORD");
$BW_ACCOUNT_ID = getenv("BW_ACCOUNT_ID");

$config = new BandwidthLib\Configuration(
    array(
        'voiceBasicAuthUserName' => $BW_USERNAME,
        'voiceBasicAuthPassword' => $BW_PASSWORD,
    )
);
$client = new BandwidthLib\BandwidthClient($config);

$voiceClient = $client->getVoice()->getClient();

$conferenceId = "c-1234";
$recordingId = "r-1234";

try {
    $response = $voiceClient->getConferenceRecording($BW_ACCOUNT_ID, $conferenceId, $recordingId);
    print_r($response->getResult()->applicationId);
} catch (BandwidthLib\APIException $e) {
    print_r($e->getResponseCode());
}
